plugins/org.eclipse.elk.core/src/org/eclipse/elk/core/util/BoxLayoutProvider.java

```diff
@@ -194,6 +194,11 @@ private KVector placeBoxes(final List<ElkNode> sortedBoxes, final double minSpac
         double stddev = areaStdDev(sortedBoxes, mean);
         
         totalArea += (sortedBoxes.size() * 1 * stddev);
+        // add top and bottom padding to total area
+        totalArea += Math.sqrt(totalArea) * (padding.getBottom() + padding.getTop());
+        // add left and right padding
+        totalArea += Math.sqrt(totalArea) * (padding.getRight());
```

Comment:
left padding is missing here. Does adding the padding here help with the underlying issue? I.e. is this something we want finished and merged or does it not really help?